src/dart/install.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -e
+source _common.sh
+
+# TODO: Do we need to "check_packages" here?
+
+# https://dart.dev/get-dart
+# "Perform the following one-time setup:"
+sudo apt-get update
+sudo apt-get install apt-transport-https
```

Comment:
This script is expected by root user, so `sudo` is not needed here.
And, should not use `sudo` because of the defarrent behavior to `su` behind proxy.

Reply:
🤦‍♀️ my bad I forgot that. Thanks!